apollo-router/src/router_factory.rs

```diff
@@ -148,6 +141,15 @@ impl SupergraphServiceConfigurator for YamlSupergraphServiceFactory {
     }
 }
 
+fn is_deduplicate_query_enabled(conf: &Configuration) -> bool {
```

Comment:
maybe that function should live in the traffic shaping plugin instead

Reply:
You're right